cmd/istio_ca/main.go

```diff
@@ -82,12 +86,14 @@ func init() {
 	flags.StringVar(&opts.signingKeyFile, "signing-key", "", "Specifies path to the CA signing key file")
 	flags.StringVar(&opts.rootCertFile, "root-cert", "", "Specifies path to the root certificate file")
 
-	flags.StringVar(&opts.namespace, "namespace", "",
-		"Select a namespace for the CA to listen to. If unspecified, Istio CA tries to use the ${"+namespaceKey+"} "+
-			"environment variable. If neither is set, Istio CA listens to all namespaces.")
+	flags.StringVar(&opts.namespace, "namespace", "", "The namesapce that CA is running at.")
+
```

Comment:
Doesn't this break e2e ?

Reply:
Shouldn't this default to istio-system ?

Reply:
Actually we don't have to, since it will check the env variable about the namespace (see line 127).

Reply:
It's reverted.